src/NextGen/ServiceProvider.php

```diff
@@ -42,9 +66,78 @@ public function boot()
             $registrar->registerGateway(PayPalStandardGateway::class);
         });
 
+        /**
+         * This module will eventually live in give-recurring
+         */
+        if (defined('GIVE_RECURRING_VERSION') && GIVE_RECURRING_VERSION) {
+            add_filter(
+                sprintf("givewp_gateway_%s_subscription_module", NextGenStripeGateway::id()),
+                static function () {
+                    return NextGenStripeGatewaySubscriptionModule::class;
+                }
+            );
+        }
+
+        give(LegacyStripeAdapter::class)->addDonationDetails();
+
+        $this->addStripeWebhookListeners();
+    }
+
+    /**
+     * @since 0.1.0
+     * @throws Framework\FormDesigns\Exceptions\OverflowException
+     */
+    private function registerFormDesigns()
+    {
         add_action('givewp_register_form_design', static function (FormDesignRegistrar $formDesignRegistrar) {
             $formDesignRegistrar->registerDesign(ClassicFormDesign::class);
             $formDesignRegistrar->registerDesign(DeveloperFormDesign::class);
         });
     }
+
+    /**
+     * @unreleased
+     */
+    private function addStripeWebhookListeners()
+    {
+//        Hooks::addAction(
+//            'give_recurring_stripe_process_invoice_payment_succeeded',
```

Comment:
This seems to be redundant anyway and  can be removed.

Reply:
resolved 0d390b5c